test/e2e/flowaggregator_test.go

```diff
@@ -259,13 +266,35 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						t.Errorf("Record does not have correct egressNetworkPolicyNamespace")
 					}
 				}
-				// Check the bandwidth using octetDeltaCount in data record.
 				if checkBandwidth {
-					checkBandwidthFromRecord(t, record, bandwidth)
+					TotalBytesFromSource = math.Max(getTotalBytesFromRecord(t, record), TotalBytesFromSource)
 				}
```

Comment:
I feel like you don't really need to condition this on `checkBandwidth` - there is no harm in always calling this function

Reply:
Thanks, addressed.